File: evaluation/cases/holdout-java-search/output.java
class Search {
    static int firstLong(String[] words, int length) {
        int result = -1;

        for (int i = 0; i < words.length; i++) {
            String word = words[i];

            if (word.length() >= length) {
                result = i;

                break;
            }
        }

        return result;
    }
}

File: evaluation/validation/php-laravel-replace-array/output.php
<?php
class ValidationSample {
    public static function replaceArray($search, $replace, $subject)
    {
        if ($replace instanceof Traversable) {
            $replace = iterator_to_array($replace);
        }

        $segments = explode($search, $subject);
        $result = array_shift($segments);

        foreach ($segments as $segment) {
            $result .= self::toStringOr(array_shift($replace) ?? $search, $search).$segment;
        }

        return $result;
    }
}
